src/main/scala/KotlinTest.scala

```diff
@@ -0,0 +1,41 @@
+package sbt
+
+import sbt.Keys._
+import sbt.classfile.Analyze
+import sbt.classpath.ClasspathUtilities
+import sbt.inc.{Analysis, IncOptions, IncrementalCompile}
+import xsbti.compile.SingleOutput
+
+object KotlinTest {
+  val kotlinTests = Def.task {
+    val old = (definedTests in Test).value
```

Comment:
I see why `:=` was used in the settings. I'd rather this switch to ignoring old and using `++=` instead.

Reply:
Yes, you are right, It was my first version ^^